Makefile

```diff
@@ -94,3 +95,10 @@ clean-all:
 clean:
 	./scripts/clean.sh
 
+# deploy services
+deploy-services:
+	./scripts/deploy-service.sh
+	
+# deploy services
+delete-services:
+	./scripts/delete-service.sh
```

Comment:
New line at the end is absent:
 https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
fixed